src/style.css

```
@@ -0,0 +1,47 @@
+@font-face {
+  font-family: sans-serif;
+  src: url(fonts/Avenir-Book.ttf);
+}
+.formfield {
+  display: block;
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+.form {
+  display: block;
+  margin-bottom: 20px;
+}
+#site-search-big {
+  background-image: url(https://raw.githubusercontent.com/yaroslav-furhala/layout_search-bar-airbnb/2bf458ddfbb9ed1bb6beca1e6040ee3e75d5df10/src/images/Search.svg);
+  background-position: 10px center;
+  padding-left: 50px;
+  border-radius: 5px;
+  background-size: 2em;
+  background-repeat: no-repeat;
+  font-weight: 300;
+  width: 100%;
+  height: 40px;
+}
+#site-search-small {
+  background-image: url(https://raw.githubusercontent.com/yaroslav-furhala/layout_search-bar-airbnb/2bf458ddfbb9ed1bb6beca1e6040ee3e75d5df10/src/images/Search.svg);
+  background-position: 10px center;
+  padding-left: 30px;
+  border-radius: 5px;
+  background-size: 1em;
+  background-repeat: no-repeat;
+  font-weight: 300;
+  width: 100%;
+  height: 20px;
+}
+#site-search-big:focus {
+  background-color: #f00;
+}
+#site-search-big:hover {
+  background-color: #ff0;
+}
+#site-search-small:focus {
+  background-color: #f00;
+}
+#site-search-small:hover {
+  background-color: #ff0;
+}

```

write all of this though comma
```suggestion
#site-search-big:focus,
#site-search-big:hover,
#site-search-small:focus,
#site-search-small:hover {
  background-color: #ff0;
}
```